### PR TITLE
Clean some docstrings

### PR DESCRIPTION
**Pull request by orchardbirds**
Made some docstrings cleaner:

- Fixed some typos

-  Made argument formatting more consistent throughout the repo

- Added types where missing, e.g.:
 batch_size -> batch_size (int)

- Split some long lines over 2 lines, etc.

### CODE REVIEW

File: examples/agents/cem.py [hunk 1]
```diff
@@ -13,12 +13,13 @@ def cem(f, th_mean, batch_size, n_iter, elite_frac, initial_std=1.0):
...
-    n_iter: number of batches
-    elite_frac: each batch, select this fraction of the top-performing samples
-    initial_std: initial standard deviation over parameter vectors
+    Args:
+        f: a function mapping from vector -> scalar
+        th_mean (np.array): initial mean over input distribution
+        batch_size (int): number of samples of theta to evaluate per batch
+        n_iter (int): number of batches
+        elite_frac (float): each batch, select this fraction of the top-performing samples
+        initial_std (float): initial standard deviation over parameter vectors
```
Comment (pzhokhov) [inline examples/agents/cem.py:22]: Looks good, but could you also add the description of the return values? 
Something like (feel free to rephrase)
```
returns: generator of dicts with the following values
      'ys' :  numpy array with values of function evaluated at current population
      'ys_mean': mean value of function over current population
      'theta_mean': mean value of the parameter vector over current population
   Subsequent dicts correspond to iterations of CEM algorithm
```

File: gym/core.py [hunk 2]
```diff
@@ -26,9 +26,8 @@ class Env(object):
 
     Note: a default reward range set to [-inf,+inf] already exists. Set it if you want a narrower range.
 
-    The methods are accessed publicly as "step", "reset", etc.. The
-    non-underscored versions are wrapper methods to which we may add
-    functionality over time.
+    The methods are accessed publicly as "step", "reset", etc...
+    The non-underscored versions are wrapper methods to which we may add functionality over time.
```
Comment (pzhokhov) [inline gym/core.py:30]: the part about non-underscored methods is obsolete and can be removed.

File: gym/envs/classic_control/cartpole.py [hunk 3]
```diff
@@ -13,17 +13,18 @@
 class CartPoleEnv(gym.Env):
     """
     Description:
-        A pole is attached by an un-actuated joint to a cart, which moves along a frictionless track. The pendulum starts upright, and the goal is to prevent it from falling over by increasing and reducing the cart's velocity.
+        A pole is attached by an un-actuated joint to a cart, which moves along a frictionless track. The pendulum
```
Comment (pzhokhov) [inline gym/envs/classic_control/cartpole.py:16]: what's the line wrap limit (max line length) here?

Comment (orchardbirds) [reply gym/envs/classic_control/cartpole.py:16]: Mine is set to 120 - just pushed to make this consistent.

File: examples/agents/cem.py (hunk 1 above)
Comment (orchardbirds) [reply examples/agents/cem.py:22]: done

File: gym/core.py (hunk 2 above)
Comment (orchardbirds) [reply gym/core.py]: done

**Comment by justinkterry** [comment]
Can this be merged?

**Pull request merged**